gateway/webhook/webhookTransformer.go

```diff
@@ -83,7 +86,7 @@ func (bt *batchWebhookTransformerT) transform(events [][]byte, sourceType string
 			}
 		]
 	*/
-	var responses []interface{}
+	var responses []transformerResponseT
```

Comment:
it would be useful to include this new scenario as a message sample in the comment above

Reply:
Done